DataDog/integrations-core
# [ETCD]  Ensure only one service check per base URL

### What does this PR do?

Modifies the check so that there is only one service check per base URL specified in the yaml file. 

### Motivation

Currently the check can submit 4 service checks per host. One for the base URL, and one for each of the three other URLs we try to hit. The issue here is that right now we currently only submit an UP/DOWN service check for the base URL. For the remaining URLs, we never submit an UP service check, only a DOWN check if we can't connect. So the DOWN Check will just stay in the UI for 24 hours. 

Instead, we submit a single service check for all endpoints, so it any one fails, the check is DOWN, then if they are ALL up, we go back to UP

### Testing Guidelines

An overview on [testing](https://github.com/DataDog/dd-agent/blob/master/tests/README.md)
is available in our contribution guidelines.

Modified the bad config test to use the base URL (the one specified in the config) as the url tag for the service check. 

### Versioning

  for the new section.

### Additional Notes

Anything else we should know when reviewing?
